virttest/iscsi.py

```diff
@@ -335,7 +335,7 @@ def logout(self):
         if self.logged_in():
             iscsi_logout(self.target)
 
-    def cleanup(self):
+    def cleanup(self, comfirmed=False):
         """
         Clean up env after iscsi used.
```

Comment:
comfirmed --> confirmed
Moreover, add doc string for this parameter

Reply:
Done